src/api-reference/expense/travelallowance/travelallowance-days.md

```diff
@@ -0,0 +1,192 @@
+* [401 Unauthorized](https://tools.ietf.org/html/rfc7235#section-3.1)
+* [403 Forbidden](https://tools.ietf.org/html/rfc7231#section-6.5.3)
+* [404 Not Found](https://tools.ietf.org/html/rfc7231#section-6.5.4)
+* [405 Method Not Allowed](https://tools.ietf.org/html/rfc7231#section-6.5.5)
+* [408 Request Timeout](https://tools.ietf.org/html/rfc7231#section-6.5.7)
+* [500 Internal Server Error](https://tools.ietf.org/html/rfc7231#section-6.6.1)
+* [502 Bad Gateway](https://tools.ietf.org/html/rfc7231#section-6.6.3)
+* [503 Service Unavailable](https://tools.ietf.org/html/rfc7231#section-6.6.4)
+* [504 Gateway Timeout](https://tools.ietf.org/html/rfc7231#section-6.6.5)
+
+#### Headers
+
+* `concur-correlationid` is a Concur specific custom header used for technical support in the form of a [RFC 4122 A Universally Unique IDentifier (UUID) URN Namespace](https://tools.ietf.org/html/rfc4122)
+* [RFC 7231 Content-Type](https://tools.ietf.org/html/rfc7231#section-3.1.1.5)
+* [RFC 7231 Content-Encoding](https://tools.ietf.org/html/rfc7231#section-3.1.2.2)
+* [RFC 7231 Content-Language](https://tools.ietf.org/html/rfc7231#section-3.1.3.2)
+* [RFC 7231 Content-Location](https://tools.ietf.org/html/rfc7231#section-3.1.4.2)
+* [RFC 7230 Content-Length](https://tools.ietf.org/html/rfc7230#section-3.3.2)
+
+#### Payload
+
+* [Schema One](#schema-one)
+* [Error](#schema-error)
+
+### <a name="details-of-travelallowance-days"></a>Get Details of TravelAllowance Days
+
+#### Request
+
+```shell
+GET https://katre-report-service-banaras.us-west-2.nonprod.cnqr.deliveryapi/api/v4/
+travelallowance/allowancedays/companyUUID/{companyUUID}/contexts/{Context}/{ContextId}
+Accept: application/json
+```
+```json
+{
+  "companyUUID": "d5528866-51b1-40a1-bab2-76296a106dcd",
+  "Context": "EXPENSE_REPORT",
+  "ContextId": "B769E8B106C04F30AE27"
+}
+```
+
+#### Response
+
+```shell
+HTTP/1.1 200 OK
+Cache-Control: max-age=604800
+Content-Type: application/json
+Date: Wed, 06 Jul 2020 17:33:03 GMT
+Etag: "359670651"
+Expires: Wed, 13 Jul 2020 17:33:03 GMT
+Last-Modified: Fri, 09 Aug 2020 23:54:35 GMT
+Content-Length: 1270
+```
+
+```json
+{
+  "content": [
+    {
+      "ExpenseTypeCode": "MEALS",
+      "FullAllowanceDay": "True",
+      "AllowanceDate": "2018-09-244T12:00:00",
+      "ExpenseTransactionDate": "2018-09-244T12:00:00",
+      "ExpenseTransactionAmount": 29
+    },
+    {
+      "ExpenseTypeCode": "MEALS",
+      "FullAllowanceDay": "False",
+      "AllowanceDate": "2018-09-247T12:00:00",
+      "ExpenseTransactionDate": "2018-09-247T12:00:00",
+      "ExpenseTransactionAmount": 29
+    }
+  ]
+}
+```
+
+## <a name="schema"></a>Schema
+
+### <a name="schema-one"></a>Get TravelAllowance Days Response Schema
+
+Name|Type|Format|Description
+---|---|---|---
+`ExpenseTypeCode`|`string`|-|Expense Code. eg: MEALS for TravelAllowance.
+`FullAllowanceDay`|`string`|-|Y/N. 'Y' means full allowance day and 'N' means partial allowance day.
```

Comment:
This description says the values are Y/N, however the example provided has True and False. Which one is correct?